services/parser-database/connector.py

```diff
@@ -61,6 +102,34 @@ def get_articles_that_match_keywords(keywords_list):
     return matching_articles
 
 
+def get_articles_by_tfidf_value(keywords_list):
```

Comment:
This is using the in_memory storage which needs to be changed to db storage just like get_articles_that_match_keywords

Reply:
Still in implementation phase.